src/tools/msccl-algorithms/ap2ll.xml

```diff
@@ -0,0 +1,2946 @@
+<algo name="allreduce_pairs" proto="LL" nchannels="2" nchunksperloop="128" ngpus="8" coll="allreduce" inplace="1" minBytes="512" maxBytes="40960">
```

Comment:
1) Need "outofplace=0" as well.
2) Need comments for MIT copyright for all XML files as well. Please refer to the first two lines in https://github.com/ROCmSoftwarePlatform/rccl/blob/develop/tools/msccl-algorithms/allreduce-allpairs-16n-16tb.xml

Reply:
updated according to the review comments.